packages/blanket/src/Blanket.tsx

```diff
@@ -1,23 +1,21 @@
 import React from 'react';
 import { withTheme } from 'styled-components';
 
-import { Root } from './styled';
+import { Root } from './styles';
```

Comment:
Why you renamed 'styled' to 'styles', if we use 'styled-components'? 'styled' (or maybe 'components') is normal name